web_responsive_pwa/static/src/js/pwa_install.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('web_responsive_pwa.systray.install', function (require) {
+"use strict";
+
+var core = require('web.core');
+var session = require('web.session');
+var SystrayMenu = require('web.SystrayMenu');
+var Widget = require('web.Widget');
+var QWeb = core.qweb;
+
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', () => {
+    navigator.serviceWorker.register('/service-worker.js')
+        .then((reg) => {
+          console.log('Service worker registered.', reg);
+        });
+  });
+}
+
+let deferredInstallPrompt = null;
+
+var PwaInstall = Widget.extend({
+    name: 'pwa_install',
+    template:'web_responsive_pwa.systray.install',
+    events: {
+    },
+    start: function () {
+        window.addEventListener('beforeinstallprompt', this.saveBeforeInstallPromptEvent);
+        document.getElementById('pwa_install_button').addEventListener('click', this.installPWA);
+        return this._super();
```

Comment:
```suggestion
        return this._super.apply(this, arguments);
```

Reply:
Hi @Yajo. Could you elaborate why this syntax is better ? just a naive question. I see that syntax (``this._super()``) in some OCA / Odoo modules

Reply:
Yes, of course :) 

It's the standard. It calls super() with any arguments that came in. Avoids the reader to have to think if super() needs args and this is a bug or not.

Reply:
https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines
...
Always use this._super.apply(this, arguments);
....

Reply:
Done, thanks